### PR TITLE
Don't check contrast for skimage save function.

### LINKED ISSUE #511: Don't check for contrast when saving images
**Is your feature request related to a problem? Please describe.**
Currently, when saving images, skimage throws a warning about low contrast for basically every image. These are annoying, especially when debugging, as it makes it hard to figure out which asserts failed

**Describe the solution you'd like**
Either 1) add a `check_contrast=False` flag to every instance of io.imsave, or change the default behavior globally to not check for low contrast images.

### PR DESCRIPTION
**Pull request by srivarra**
- add an ignore_warnings function for imageio.core.util
- added reference

**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Closes #511. Removes the low contrast warnings for images.

**How did you implement your changes**

Manually added `check_contrast=False` to all instances of `skimage.io.imsave` in ark.

In addition fixed a small typo and added a line to the `.gitignore` to ignore
`.python-version` files (pyenv).

[Reference](https://stackoverflow.com/questions/52486237/how-to-turn-off-skimage-warnings)

**Remaining issues**

None at the moment.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -21,3 +21,4 @@ tables>=3.6.1,<4
 umap-learn>=0.4.6,<1
 xarray>=0.12.3,<1
 tqdm>=4.54.1,<5
+imageio>=2.3.0
```
Comment (ngreenwald) [inline requirements.txt:24]: Do we still need this?

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Looks good. I assume the imageio import was for the initial version that was going to define a function? Or is there a reason to still pin it.

File: requirements.txt (hunk 1 above)
Comment (srivarra) [reply requirements.txt:24]: Yeah it was initially used for a function. It's not needed now.

**Review by ngreenwald — APPROVED** [review]
Looks good. Will you make the same change for toffy and mibi_bin_tools as well?

**Comment by srivarra** [comment]
Yeah I can do that.

**Pull request merged**